package.json

```diff
@@ -42,7 +42,7 @@
   },
   "peerDependencies": {
     "tslint": "^5.0.0",
-    "typescript": ">=2.1 <=2.7"
+    "typescript": ">=2.1 <=2.8"
```

Comment:
I think `^2.1.0` would be more appropriate here, as this should work fine until the next major release of typescript.

Reply:
Normally I'd agree, but TS doesn't follow semver. They have documented breaking changes on pretty much every minor release.